index.d.ts

```diff
@@ -1,14 +1,16 @@
 import * as React from 'react';
```

Comment:
Would it be possible to change this to `import React, { ComponentType, ComponentClass } from 'react';` and get rid of the alias types?